.github/workflows/sycl-linux-precommit.yml

```diff
@@ -58,7 +58,9 @@ jobs:
       changes: ${{ needs.detect_changes.outputs.filters }}
 
       toolchain_artifact: sycl_linux_default
-      e2e_binaries_artifact: sycl_e2e_bin_default
+      e2e_binaries_artifact: e2e_bin
```

Comment:
again not really related but i wonder if we really need to expose as inputs to `sycl-linux-build`, could we just require they are `e2e_bin{_spirv_backend, _preview}`? i don't expect these will ever be used by anyone outside of CI testing so the name shouldnt matter to anyone

Reply:
What I'm trying to do is to allow specifying only some kinds of e2e tests vs all-or-nothing (I'm not interested in preview mode when creating a container image for the compatibility testing). 

It doesn't make much difference between three `bool`s with default names vs three explicit names. However, we do need to pass the names later on to `sycl-linux-run-tests.yml` and it's easier if the names are defined here vs defaulted inside the `sycl-linux-build.yml` workflow.